egs2/chime7_task1/asr1/run.sh

```diff
@@ -80,9 +80,15 @@ asr_batch_size=$(calc_int 128*$ngpu) # reduce 128 bsz if you get OOMs errors
 asr_max_lr=$(calc_float $ngpu/10000.0)
 asr_warmup=$(calc_int 40000.0/$ngpu)
 
-if [ $decode_only -eq 1 ]; then
+if [ $decode_only == "dev" ]; then
   # apply gss only on dev
   gss_dsets="chime6_dev,dipco_dev,mixer6_dev"
+  asr_tt_set="kaldi/chime6/dev/gss kaldi/dipco/dev/gss/ kaldi/mixer6/dev/gss/"
+elif
+  [ $decode_only == "eval" ]; then
+  # apply gss only on dev
```

Comment:
The comment should be updated as `eval`, right?

Reply:
updated